src/linear/updater_coordinate.cc

```diff
@@ -40,6 +40,7 @@ class CoordinateUpdater : public LinearUpdater {
     FromJson(config.at("coordinate_param"), &cparam_);
   }
   void SaveConfig(Json* p_out) const override {
+    LOG(INFO) << "Save config for CPU updater.";
```

Comment:
Please remove this line

Reply:
I use it for testing, will change it into log debug

Reply:
done.